lib/components/narrative/default/default-itinerary.js

```diff
@@ -146,7 +111,8 @@ const ITINERARY_ATTRIBUTES = [
       return (
         // FIXME: For CAR mode, walk time considers driving time.
         <span>
-          <FormattedDuration duration={itinerary.walkTime} />{' '}
+          <FormattedDuration duration={itinerary.walkTime} />
+          {' '}
```

Comment:
I think adding whitespace is a no-no when using internationalization. If this is for styling purposes to add padding or something, the styling should be changed instead.

Reply:
I believe this is for formatting purposes (the white space between time and icon)
![image](https://user-images.githubusercontent.com/63798641/129950740-c848ec9a-e1e3-4d83-b8f3-849c30c68578.png)
I'm not sure how to go about changing this via styling though, would it require a special LegIcon component?

Reply:
Addressed in https://github.com/opentripplanner/otp-react-redux/pull/430/commits/8297d2e4113967e47f5abaed0762f8b204793144.